packages/account/src/Components/sample-credit-card-modal/__tests__/sample-credit-card-modal.spec.tsx

```diff
@@ -0,0 +1,36 @@
+import React from 'react';
+import { screen, render } from '@testing-library/react';
+import { SampleCreditCardModal } from '../sample-credit-card-modal';
+
+describe('SampleCreditCardModal', () => {
+    let modal_root_el: HTMLDivElement;
+    beforeAll(() => {
+        modal_root_el = document.createElement('div');
+        modal_root_el.setAttribute('id', 'modal_root');
+        document.body.appendChild(modal_root_el);
+    });
+
+    afterAll(() => {
+        document.body.removeChild(modal_root_el);
+    });
+
+    it('should render modal props', () => {
+        const props = {
+            is_open: true,
+            onClose: jest.fn(),
+        };
+        render(<SampleCreditCardModal {...props} />);
+        expect(screen.getByRole('heading')).toHaveTextContent('How to mask your card?');
+        expect(screen.getByRole('img')).toHaveAttribute('alt', 'creditcardsample');
+    });
+
+    it('should not render modal when is_open is false', () => {
+        const props = {
```

Comment:
```suggestion
        const props: React.ComponentProps<typeof SampleCreditCardModal>  = {
```
maybe we could define props in outer scope and just change the field for this test case?